modules/game/src/main/Handicaps.scala

```diff
@@ -69,6 +73,18 @@ object Handicaps {
     computeRankDiff(rating, diff, 0)
   }
 
+  def convertGoRating(goRating: String): Int =
+    goRating match {
+      case x if x.matches("""\d+k""") =>
```

Comment:
I would prefer the use of proper regexes here like we do in SG to match on UCI strings. But I dont feel strongly enough to force you to change this now